### PR TITLE
Support custom tags (#1710)

### PR DESCRIPTION
**Pull request by BafS**
Fix https://github.com/nelmio/NelmioApiDocBundle/issues/1710

### CODE REVIEW

File: OpenApiPhp/Util.php [hunk 1]
```diff
@@ -374,6 +374,12 @@ private static function mergeFromArray(OA\AbstractAnnotation $annotation, array
                 $propertyName = 'ref';
             }
             if (!\in_array($propertyName, $done, true)) {
+                // Specification Extensions using custom tags (starting with `x-`)
+                if (2 < strlen($propertyName) && 'x_' === substr($propertyName, 0, 2)) {
```
Comment (GuilhemN) [inline OpenApiPhp/Util.php:378]: They actually begin with `x-` :)

**Comment by GuilhemN** [comment]
Thank you for submitting this PR!
Could you add a small test to this PR?

**Comment by mvanduijker** [comment]
Would love to see this merged.

@BafS do you need any help to complete this?

**Comment by GuilhemN** [comment]
Feel free to open a new PR if you'd like to finish this @mvanduijker :)

**Comment by DjordyKoert** [comment]
Fixed in https://github.com/nelmio/NelmioApiDocBundle/pull/2230

**Pull request closed without merging**